tendermint/src/lite/verifier.rs

```diff
@@ -184,98 +124,120 @@ where
     Ok(())
 }
 
-/// Returns Ok if we can trust the passed in (untrusted) header
-/// based on the given trusted state, otherwise returns an Error.
-fn can_trust_bisection<TS, SH, C, L, S, R>(
-    trusted_state: &TS,    // h1 in spec
-    untrusted_header: &SH, // h2 in spec
+// Verify a single untrusted header against a trusted state.
+// Includes all validation and signature verification.
+// Not publicly exposed since it does not check for expiry
+// and hence it's possible to use it incorrectly.
+// If trusted_state is not expired and this returns Ok, the
+// untrusted_sh and untrusted_next_vals can be considered trusted.
```

Comment:
These comments should be proper doc strings prefixed with `///`.

Reply:
This function isn't supposed to be exposed which is why they're not.

It's marked `pub` because it's needed in the tests, but it shouldn't be exposed in the `lite.rs` defining the public API for the crate. Of course currently it is because it's used in an integration test, but fixing that is tracked separately.